spec/services/create_user_spec.rb

```diff
@@ -1,81 +1,37 @@
 require "rails_helper"
 
-RSpec.describe CreateUser, pending: "To be finished as part of the user invite card" do
-  let(:user) { build(:administrator) }
-  before(:each) do
-    stub_auth0_token_request
-  end
+RSpec.describe CreateUser do
+  let(:user) { build(:administrator, :new_user) }
 
   describe "#call" do
     it "returns a successful result" do
-      stub_auth0_create_user_request(email: user.email)
-
-      result = described_class.new(user: user, organisation: build_stubbed(:delivery_partner_organisation)).call
+      result = CreateUser.new(user: user, organisation: build_stubbed(:delivery_partner_organisation)).call
 
       expect(result.success?).to eq(true)
-      expect(result.failure?).to eq(false)
     end
 
     it "sends a welcome email to the user" do
-      stub_auth0_create_user_request(email: user.email)
-
       expect {
-        described_class.new(user: user, organisation: build_stubbed(:delivery_partner_organisation)).call
+        CreateUser.new(user: user, organisation: build_stubbed(:delivery_partner_organisation)).call
       }.to have_enqueued_mail(UserMailer, :welcome).with(user)
     end
 
+    it "creates a default password, known to no-one" do
+      CreateUser.new(user: user, organisation: build_stubbed(:delivery_partner_organisation)).call
+      expect(User.last.encrypted_password).not_to be_blank
```

Comment:
Feels mildly wierd to depend on the new user being the last one -- seems fine, though.

Reply:
It felt weird to me too! We can rely on the testing env being artificial conditions though, where we know that's the only email that's going to be sent.